templates/nextjs/fuels.config.ts

```diff
@@ -10,7 +10,7 @@ const fuelCorePort = +(process.env.NEXT_PUBLIC_FUEL_NODE_PORT as string) || 4000
 
 export default createConfig({
   workspace: './sway-programs',
-  output: './src/sway-api',
+  output: './sway-api',
```

Comment:
overall, I changed structure and moved all pages to app folder because for me it seems standard practice to have separate folder for all routes in nextjs. I moved everything not related to routing (sway-api, styles) layer upper to have them separated. I used underscore in components folders to distinguish them from routes folders. I used relative path imports because most of those components are used only inside "app" folder, so here is no need to use alias. I may be wrong, please correct me if those changes does not needed, I would appreciate any additional knowledge that can improve my PR

Reply:
@petertonysmith94 hello, I am closing this PR, will create another one with more structured changes